lms/djangoapps/certificates/models.py

```diff
@@ -130,6 +130,8 @@ class CertificateWhitelist(models.Model):
     regardless of their grade unless they are on the
     embargoed country restriction list
     (allow_certificate set to False in userprofile).
+
+    .. no_pii: It is possible that information here is identifiable, but certificates aren't retired so we need this.
```

Comment:
Why not a positive annotation with a retirement type of "retained" if you acknowledge that this possibly contains PII?

Reply:
This doesn't actually contain any of what we define as PII, so I can fix that comment.